### ISSUE TITLE
How can I extract the doc comments in the XML file ?

### ISSUE THREAD

**Issue by C-far** [pawn-lang/YSI-Includes#61]
Hi,
I've seen the docs in the files before each function and I've seen it is possible to get a XML file.
When we open this XML file, we see by example this : http://i.imgur.com/8svrBPb.png (Guide Pawn pdf)

So, how can I do this ?
Regards.

**Comment by Crayder** [comment]
They are in XML format already... And they are all in 'documentation' comments (they are wrapped in comments that use two asterisks instead of just one). Parse and extract those comments.

(I think the pawn toolkit has a parser that does this for you, if I remember correctly)

EDIT: Scratch that, I want this too... XD

**Comment by Crayder** [comment]
Figured it out.

First download the pawn toolkit and put the xml folder with your compiler. Add the '-r' option to the compiler and it'll output an xml. Open that xml and you have it.

**Comment by Ahmad45123** [comment]
I and @Crayder are working on extracting all the docs to the wiki by the way.
In case that's what you want.

**Comment by Crayder** [comment]
Well that's not directly related, since I actually planned on doing that before (I have proof, just read the current wiki home page XD).

**Comment by C-far** [comment]
It's weird, 'cause I managed to have the rendering of the XML file, but only by : 
- placing the XML file in the folder pawno\xml.
- editing the XML file and the XLS fle at the lines "<?xml-stylesheet ?>" (for XML) and "<LINK rel="stylesheet" /> (for XLS).

**Comment by Crayder** [comment]
Yeah I had to edit the xsl path inside my xml, but after that it worked perfectly fine.
